application/templates/alertmanagerconfig.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.alertmanagerConfig -}}
+{{- if .Values.alertmanagerConfig.enabled -}}
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: {{ template "application.name" . }}
+  namespace: {{ template "application.namespace" . }}
+  labels:
```

Comment:
Are these labels mandatory? if not then,please add if condition for that

Reply:
Labels are mandatory. as AlertmanagerConfig is picked and merged by matching the labels